[2024-08-07T14:20:10.032+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T14:20:10.055+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-space-exploration.Autorizacion_NASA manual__2024-08-07T14:20:08.695612+00:00 [queued]>
[2024-08-07T14:20:10.064+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-space-exploration.Autorizacion_NASA manual__2024-08-07T14:20:08.695612+00:00 [queued]>
[2024-08-07T14:20:10.065+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-07T14:20:10.081+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): Autorizacion_NASA> on 2024-08-07 14:20:08.695612+00:00
[2024-08-07T14:20:10.087+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=261) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T14:20:10.089+0000] {standard_task_runner.py:64} INFO - Started process 262 to run task
[2024-08-07T14:20:10.089+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***-space-exploration', 'Autorizacion_NASA', 'manual__2024-08-07T14:20:08.695612+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmp90ccgsjj']
[2024-08-07T14:20:10.090+0000] {standard_task_runner.py:91} INFO - Job 23: Subtask Autorizacion_NASA
[2024-08-07T14:20:10.137+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow-space-exploration.Autorizacion_NASA manual__2024-08-07T14:20:08.695612+00:00 [running]> on host 38873943cc86
[2024-08-07T14:20:10.233+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-space-exploration' AIRFLOW_CTX_TASK_ID='Autorizacion_NASA' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T14:20:08.695612+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-07T14:20:08.695612+00:00'
[2024-08-07T14:20:10.235+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T14:20:10.250+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T14:20:10.250+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "sleep 20 && echo 'OK' > /tmp/response_20240807.txt"]
[2024-08-07T14:20:10.262+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T14:20:30.266+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T14:20:30.267+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T14:20:30.290+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***-space-exploration, task_id=Autorizacion_NASA, run_id=manual__2024-08-07T14:20:08.695612+00:00, execution_date=20240807T142008, start_date=20240807T142010, end_date=20240807T142030
[2024-08-07T14:20:30.310+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T14:20:30.333+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T14:20:30.336+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
